frontend/translations/ukrainian.js

```diff
@@ -0,0 +1,80 @@
+const data = {
+  'Selected': 'Вибрано: {0} з {1}',
+  'Uploading files': 'Завантаження {0}% із {1}',
+  'File size error': '{0} завеликий, будь ласка, завантажте файл менший за {1}',
+  'Upload failed': '{0} не вдалося завантажити',
+  'Per page': '{0} На сторінку',
+  'Folder': 'Папка',
+  'Login failed, please try again': 'Вхід не виконано. Будь ласка, спробуйте ще раз',
+  'Already logged in': 'Вже авторизовано.',
+  'Please enter username and password': 'Будь ласка, введіть ім'я користувача та пароль.',
```

Comment:
I am sure ' needs to be escaped like \' so it will not break

Reply:
No javascript requires escaping it won't be visible on the web page.. 

https://stackoverflow.com/questions/2004168/escape-quotes-in-javascript

Reply:
And even Github removes \ infront of ```\'``` great..

Reply:
> And even Github removes \ infront of `\'` great..

I have corrected the translation.
-
Sorry, I don't know English very well. Perhaps due to the difficulties of Google Translate, we misunderstood each other.